README.md

```diff
@@ -62,10 +62,10 @@ Check out [plugins documentation](./docs/plugins.md).
 First, clone the repo via git:
 
 ```bash
-git clone https://github.com/KELiON/cerebro.git cerebro
+$ git clone https://github.com/KELiON/cerebro.git cerebro
```

Comment:
@KELiON Linux doesn't behave any differently from macOS here. @lex111 only added it for indicating that it is a command.